docs/self-managed/tasklist-deployment/taskslist-api.md

```diff
@@ -0,0 +1,21 @@
+---
+id: tasklist-api
+title: Tasklist API
+---
+
+Tasklist offers an API (GraphQL) where clients authenticate via token. Therefore, users can use the API and integrate it into their own systems without having to use Tasklist UI.
+
+## Self-Managed API
+
+As there are different authentication methods available for the Self-Managed Tasklist, we don't offer the machine-to-machine (m2m) token. The current approach requires the following:
+
+- Clients must create a login request: `POST /api/login?username=<USERNAME>&password=<PASSWORD>`.
+- Use response headers/cookies in the API GraphQL.
+
+### Limitations
+
+As in current Self-Managed versions, we can't differentiate a regular `User` request versus an `API User` request; this imposes some limitations to the Self-Managed Tasklist API:
+
+- Tasks must be claimed by `API User` before any changes are made.
+- Tasks must be assigned to `API User` itself. You cannot assign these tasks to someone else.
+- Tasks can **only** be completed if previously claimed/assigned to `API User`.
```

Comment:
Is this the same note as the first one claiming "Tasks must be claimed by `API User` before any changes are made."?

Reply:
@santanaluiz Can you clarify this so we can merge? 🙂

Reply:
changes can be adding a variable for example. Completed is another action.

Sorry for the delay on this. I was on vacation :)